setup.py

```diff
@@ -121,6 +121,9 @@ def get_cmake_path():
     raise Exception("CMake must be installed to build from source.")
 
 
+use_openssl = os.environ.get('AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO', False)
```

Comment:
Let's try and use the term "use system libcrypto" as much as possible, vs the kinda misleading "use openssl" term

Also, this file tends to uses functions over global variables, so let's stick with that

Also, this line weirdly mixes strings and booleans (`use_openssl` could end up as False or "" or "0" or "1")

maybe:
```
def using_system_libcrypto():
    # if we're going ifdef style
    return 'AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO' in os.environ
    # or if we're going set-to-1 style   
    return os.environ.get(AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO) == "1"
```